lib/Backend/BailOut.cpp

```diff
@@ -252,10 +251,19 @@ BailOutInfo::FinalizeBailOutRecord(Func * func)
             }
             // The locals size contains the inlined-arg-area size, so remove the inlined-arg-area size from the
             // adjustment for normal locals whose offsets are relative to the start of the locals area.
-            offset -= (localsSize - inlineeArgStackSize);
+            offset -= (inlinedArgSlotAdjust - inlineeArgStackSize);
```

Comment:
Sorry, my bad. I missed that macro.

---
In reply to: [81579468](https://github.com/Microsoft/ChakraCore/pull/1641#discussion_r81579468) [](ancestors = 81579468)